pages/docs/v3/updating/3-0.md

```diff
@@ -256,6 +256,41 @@ In `ios/App/App/AppDelegate.swift`, update the following:
      }
 ```
 
+### Remove USE_PUSH macro
+
+If using the push notifications feature, in `ios/App/App/AppDelegate.swift`, update the following:
```

Comment:
It's not at all clear why they might need to do this so I think providing some context would help.
```suggestion
If you are using push notifications, the recommendation is to remove the `USE_PUSH` compilation condition in `ios/App/App/AppDelegate.swift` to guarantee that the conditional code is included. Otherwise you will need to update the project's build settings to ensure the correct flag gets passed to the compiler. Update the following:
```

Reply:
Since this is an upgrade guide, the compilation option should be there as it was included by default on the template.